server/models/user.coffee

```diff
@@ -74,15 +76,25 @@ User.getUsername = (callback) ->
 
```

Comment:
Good point! This part can be better now for the new onboarding. Thanks @gregorylegarec. It's fixed. 👍